iib/web/migrations/versions/7573241a5156_rename_bundle_deprecation_association_.py

```diff
@@ -0,0 +1,70 @@
+"""Rename bundle deprecation association for merge index.
+
+Revision ID: 7573241a5156
+Revises: eec630370e68
+Create Date: 2021-02-09 13:50:01.905796
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '7573241a5156'
+down_revision = 'eec630370e68'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'request_merge_bundle_deprecation',
+        sa.Column('merge_index_image_id', sa.Integer(), autoincrement=False, nullable=False),
+        sa.Column('bundle_id', sa.Integer(), autoincrement=False, nullable=False),
+        sa.ForeignKeyConstraint(['bundle_id'], ['image.id'],),
+        sa.ForeignKeyConstraint(['merge_index_image_id'], ['request_merge_index_image.id'],),
+        sa.PrimaryKeyConstraint('merge_index_image_id', 'bundle_id'),
+        sa.UniqueConstraint(
+            'merge_index_image_id', 'bundle_id', name='merge_index_bundle_constraint'
+        ),
+    )
+    with op.batch_alter_table('request_merge_bundle_deprecation', schema=None) as batch_op:
+        batch_op.create_index(
+            batch_op.f('ix_request_merge_bundle_deprecation_bundle_id'), ['bundle_id'], unique=False
+        )
+        batch_op.create_index(
+            batch_op.f('ix_request_merge_bundle_deprecation_merge_index_image_id'),
+            ['merge_index_image_id'],
+            unique=False,
+        )
+
+    with op.batch_alter_table('bundle_deprecation', schema=None) as batch_op:
+        batch_op.drop_index('ix_bundle_deprecation_bundle_id')
+        batch_op.drop_index('ix_bundle_deprecation_merge_index_image_id')
+
+    op.drop_table('bundle_deprecation')
+
+
+def downgrade():
+    op.create_table(
+        'bundle_deprecation',
+        sa.Column('merge_index_image_id', sa.INTEGER(), nullable=False),
```

Comment:
In the other `create_table` call, `sa.Integer()` is used. Should we be consistent here instead of using `sa.INTEGER()` ?